ngraph/test/onnx/onnx_import_dyn_shapes.in.cpp

```diff
@@ -1206,23 +1206,23 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_instance_normalization_dyn_shape2)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample9_scales_input_nearest_infer)
-{
-    const auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/upsample9_scales_input_nearest.prototxt"));
-
-    // Input data shape (1, 1, 2, 2)
-    // mode: nearest
-
-    Shape expected_output_shape{1, 1, 4, 6};
-    auto test_case = test::TestCase<TestEngine, TestCaseType::DYNAMIC>(function);
-    test_case.add_input<float>({1.0, 2.0, 3.0, 4.0});
-    test_case.add_input<float>({1.0, 1.0, 2.0, 3.0});
-    test_case.add_expected_output<float>(
-        expected_output_shape, {1.0, 1.0, 1.0, 2.0, 2.0, 2.0, 1.0, 1.0, 1.0, 2.0, 2.0, 2.0,
-                                3.0, 3.0, 3.0, 4.0, 4.0, 4.0, 3.0, 3.0, 3.0, 4.0, 4.0, 4.0});
-    test_case.run();
-}
+// NGRAPH_TEST(${BACKEND_NAME}, onnx_upsample9_scales_input_nearest_infer)
```

Comment:
Why do you comment this test?

Reply:
These tests were temporarily disabled to write NGRAPH reference implementation of Interpolate-4. Now I am working with these tests.

Reply:
Please resolve this issue before the merge

Reply:
Please create a ticket to enable this test

Reply:
Created #38204.